hera_qm/ant_metrics.py

```diff
@@ -609,117 +577,85 @@ def save_antenna_metrics(self, filename, overwrite=False):
+                    overwrite=False, Nbls_per_load=None, history='', verbose=True):
     """
     Run a series of ant_metrics tests on a given set of input files.
 
     Note
     ----
-    The funciton will take in a list of files and options. It will run the
-    series of ant metrics tests, and produce an HDF5 file containing the
-    relevant information. The file list need only contain one polarization
-    type for a given JD, because the function will look for the other
-    polarizations in the same folder. If not all four polarizations are found,
-    a warning is generated, because the code assumes all four polarizations are
-    present.
+    The funciton will take a file or list of files and options. It will run
+    ant metrics once on all files together but then save the results to am
+    identical HDF5 file for each input file.
 
     Parameters
     ----------
-    files : list of str
-        List of files to run ant metrics on. Can be any of the 4 polarizations.
-    pols : list of str, optional
-        List of polarizations to perform metrics over. Allowed polarizations: 'xx',
-        'yy', 'xy', 'yx'. Default is ['xx', 'yy', 'xy', 'yx'].
+    data_files : str or list of str
+        Path to file or files of raw data to calculate antenna metrics on.
+    apriori_xants : list of integers or tuples, optional
+        List of integer antenna numbers or antpol tuples e.g. (0, 'Jee') to mark
+        as excluded apriori. These are included in self.xants, but not
+        self.dead_ants or self.crossed_ants when writing results to disk.
     crossCut : float, optional
-        Modified Z-Score limit to cut cross-polarized antennas. Default is 5.0.
+            Modified Z-Score limit to cut cross-polarized antennas. Default is 5.0.
     deadCut : float, optional
         Modifized Z-Score limit to cut dead antennas. Default is 5.0.
-    alwaysDeadCut : float, optional
-        Modified Z-Score limit for antennas that are definitely dead. Antennas with
-        z-scores above this limit are thrown away before iterative flagging.
-        Default is 10.0.
+    run_cross_pols : bool, optional
+        Define if mean_Vij_cross_pol_metrics is executed. Default is True.
+    run_cross_pols_only : bool, optional
+        Define if mean_Vij_cross_pol_metrics is the *only* metric to be run.
+        Default is False.
     metrics_path : str, optional
         Full path to directory to story output metric. Default is the same directory
         as input data files.
     extension : str, optional
         File extension to add to output files. Default is ant_metrics.hdf5.
-    vis_format : str, optional
-        File format of input visibility data. Must be one of: 'miriad', 'uvh5',
-        'uvfits', 'fhd', 'ms' (see pyuvdata docs). Default is 'miriad'.
-    verbose : bool, optional
-        If True, print out statements during iterative flagging. Default is True.
+    overwrite: bool, optional
```

Comment:
It's a mix, unfortunately. `write_metric_file` uses overwrite, so that's why I used this here. However, in the script, I used clobber and then just said `overwrite=args.clobber`

I'd rather not break other APIs for other parts of hera_qm.

Reply:
That's a good reason. We can keep it.